### PR TITLE
Added Dominican Republic holidays

### PR DESCRIPTION
**Pull request by KiritoXD01**
This pull request adds Dominican Republic holidays in spanish. In the future I can add translation if needed

### CODE REVIEW

File: src/Countries/DominicanRepublic.php [hunk 1]
```diff
@@ -0,0 +1,42 @@
...
+use Carbon\CarbonImmutable;
+
+class DominicanRepublic extends Country
+{
+    public function countryCode(): string
+    {
+        return 'do';
+    }
+
+    /** @return array<string, CarbonImmutable | string> */
```
Comment (Nielsvanpach) [inline src/Countries/DominicanRepublic.php:14]: You can remove this line

**Review by Nielsvanpach — COMMENTED** [review]
Thanks! Could you rebase your branch on main, so I can run the tests here?

**Comment by Nielsvanpach** [comment]
Could you make sure all tests and phpstan are green? I think you removed one line too much

**Comment by KiritoXD01** [comment]
I just realized 😅

**Comment by KiritoXD01** [comment]
@Nielsvanpach Sorry this is my first PR maybe I'm doing something wrong

**Comment by vivgui** [comment]
hey my fellow Dominican, just wanted to point out that `'Día de la Virgen de la Altagracia' => '12-08',` is not a national holiday, only the catholic church recognizes that day but its not a holiday

**Comment by KiritoXD01** [comment]
@vivgui thanks for the clarification, I'll update it

**Comment by KiritoXD01** [comment]
@Nielsvanpach I fixed some holidays, now should be happy with the tests

**Comment by Nielsvanpach** [comment]
Thanks!

**Pull request merged**